server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -119,6 +122,124 @@ const orderMutations = {
       return models.Order.findOne(query);
     },
   },
+  updateOrder: {
+    type: Order,
+    description: "Update an Order's amount, tier, payment method, or frequency",
+    args: {
+      order: {
+        type: new GraphQLNonNull(OrderReferenceInput),
+        description: 'Object matching the OrderReferenceInput (id)',
+      },
+      amount: {
+        type: GraphQLInt,
+        description: 'Amount in cents of the order',
+      },
+      frequency: {
+        type: GraphQLString,
+        description: 'Frequency of the recurring order, either MONTHLY or YEARLY',
+      },
+      tier: {
+        type: GraphQLString,
```

Comment:
That will not play well as a String. Should be a TierReferenceInput or similar.

Reply:
Ok, noted. I won't actually get to the tier work until next week - focusing on polishing/merging add/update payment stuff this week. I was mostly 'drafting' the updateOrder mutation. Should I remove this for now or just change it next week?